extensions/standard-replies.md

```diff
@@ -50,6 +50,11 @@ Reply codes (especially more widely-applicable ones such as `NO_SUCH_NICK`) may
 
 `<description>` is a required final parameter, containing a plain-text description of the reply message. For example, for a `FAIL` message, the description should clearly state what error occurred, and how the user should act to resolve it. Implementers should write their descriptions to be easily understood by end users, for example _"Passwords must contain lowercase and uppercase letters, and be 8 or more characters"_ is a more clear description than _"Password consistency and strength enumerability (sec/* internal) check failed"_.
 
+### Capabilities
+
+This specification adds the `standard-replies` capability. This capability allows servers to send arbitrary standard replies other than ones enabled by a specific specification. It also allows clients to indicate support for receiving any well-formed standard reply, whether or not it is recognised and used.
+
+It is RECOMMENDED that servers use standard replies instead of error numerics or notices when responding to a client which supports this capability. It is NOT RECOMMENDED that servers which support this capability add new error numerics.
```

Comment:
This potentially requires standardizing error codes for all existing numerics?

Reply:
It's only a recommendation (i.e. should) and it says **new** errors numerics in the next sentence. If you have any ideas for alternative phrasing here I'd be happy to tweak it though.

Reply:
I mean, as a client dev, I really need to know what kind of error code I'd get from commands like JOIN or PRIVMSG in order to match them properly. IOW, enabling this suggested cap would regress my client without any way to un-break it. IOW, this cap is more ill-defined than error numerics.

For reference, example client code: https://git.sr.ht/~emersion/goguma/tree/master/item/lib/client.dart#L981

Reply:
BTW -- "It's only a recommendation" makes it worse IMHO because clients don't know what to expect and will need to handle all cases (both error numerics and standard replies), only increasing the number of lines of code without any other benefit.

Reply:
I just updated it. Is the new language better?

Reply:
Yes, thanks!